dispatcher/src/components/FilterMobile/SubFilterPage.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { blueArrow } from "../../assets";
+import {
+  FilterBackArrow,
+  FullDivider,
+  FilterLine,
+  FilterText,
+  FilterTitle,
+} from "./StyledFilterMobile";
+import categories from "../../data/categories.json";
+
+export interface SubFilterPageProps {
+  title?: string;
+  onBackClick: (click: boolean) => void;
+}
+
+const SubFilterPage: React.FC<SubFilterPageProps> = ({
+  title,
+  onBackClick,
+}) => {
+  return (
+    <>
+      <FilterTitle>
+        <FilterBackArrow src={blueArrow} onClick={() => onBackClick(false)} />
+        {title}
+      </FilterTitle>
+      <FullDivider />
+      {title === "Dates" && (
+        <>
+          {categories.dates.map((item, index) => (
+            <div key={index}>
+              <FilterLine>
+                <FilterText>{item.name}</FilterText>
+              </FilterLine>
+              <FullDivider />
+            </div>
+          ))}
+        </>
+      )}
+      {title === "Country" && (
+        <>
+          {categories.countries.map((item, index) => (
+            <div key={index}>
+              <FilterLine>
+                <FilterText>{item.name}</FilterText>
+              </FilterLine>
+              <FullDivider />
+            </div>
+          ))}
+        </>
+      )}
+      {title === "Languages" && (
+        <>
+          {categories.languages.map((item, index) => (
```

Comment:
Note that these lines are repeating themselves. maybe you should export them to a separate component?